docs/blog/posts/2023/nanog88-python-pygnmi.md

```diff
@@ -0,0 +1,99 @@
+---
+date: 2023-07-13
+tags:
+  - media
+  - nanog
+  - chatgpt
+  - gnmi
+  - gnmic
+  - pygnmi
+authors:
+  - mau
+---
+
+# :material-github: Finding missconfigurations in your Fabric
+
+<small>:material-github: [Git Repo](https://github.com/cloud-native-everything/pygnmi-srl-nanog88)</small>
+
+Today, I'm sharing another piece of my experience from the [NANOG88](https://www.nanog.org/events/nanog-88/) conference where I had the privilege of presenting a tutorial featuring pyGNMI, a powerful tool for diagnosing network issues. During my talk, I used pyGNMI to visualize EVPN Layer2 and Layer 3 domains, sorting them by switch or network instance. I also added a special feature that detects discrepancies in the settings between different switches using the same EVPN domain – a great way to catch typos in your BGP/VXLAN settings.
+
+For this demonstration, I leveraged containerlab and SR Linux to build a VXLAN-EVPN Fabric, replicating a typical configuration I often use in my Kubernetes labs. I incorporated eBGP for underlay communication, and the topology I utilized comprised two spines, two leaf switches, and a border leaf.
+
+Check out the full details in the corresponding [Git Repo](https://github.com/cloud-native-everything/pygnmi-srl-nanog88)
+
+## How this app works
```

Comment:
@cloud-native-everything can you also add a short section here on how to deploy the lab if users want to follow your examples?

Reply:
done